docs/tutorial-node-implementation.md

```diff
@@ -255,11 +255,20 @@ Here you can add a readme and license if you want.
 
 ![Implementation readme](./images/new_node_type/implementations/implementation_readme.png)
 
-### 2. Create Lifecycle implementation
+### 2. Supported Implementation Artifact Types
 
-Go to the _Implementation Artifacts_ tab.
-We support several _Implementation Artifact Types_ including Shell Scripts, Java Services, Python and even Ansible or Chef.
-Add the `install` artifact.
+We support several _Implementation Artifact Types_:
+
+- Ansible Playbook: Execute an Ansible playbook on the target. The playbook is automatically transferred to the target. Ansible is ensured to be installed on the target.
```

Comment:
Highlight the types by using the code element: `Ansible Playbook`. Do it for all types.

Reply:
I now use the exact artifacttype as defined in https://github.com/OpenTOSCA/container/tree/master/org.opentosca.bus/org.opentosca.bus.management.invocation.plugin.script/src/main/resources/artifacttypes